### PR TITLE
techdebt: [CDS-71663]: Removal of UI usages of GITOPS FFs

### PR DESCRIPTION
**Pull request by mankrit-singh**
### Summary

Removes FF usages - GITOPS_ORG_LEVEL, GITOPS_IAM

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by mankrit-singh** [comment]
retrigger titlecheck

**Review by ishantmehtaharness — DISMISSED** [review]
Looks Good

File: src/modules/10-common/components/ResourceCardList/ResourceCardList.tsx [hunk 1]
```diff
@@ -99,10 +98,7 @@ const ResourceCardList: React.FC<ResourceCardListProps> = ({ items }) => {
 
   const { isOpen: showGitOpsEntities, toggle: toggleShowGitOpsEntities } = useToggleOpen()
   const { loading, data } = useGetSmtpConfig({ queryParams: { accountId } })
-  let showGitOpsCard = accountId && !orgIdentifier && !projectIdentifier
-  if (GITOPS_ORG_LEVEL) {
-    showGitOpsCard = (accountId || orgIdentifier) && !projectIdentifier
-  }
+  let showGitOpsCard = (accountId || orgIdentifier) && !projectIdentifier
```
Comment (ishantmehtaharness) [inline src/modules/10-common/components/ResourceCardList/ResourceCardList.tsx:101]: const showGitOpsCard = (accountId || orgIdentifier) && !projectIdentifier

Comment (mankrit-singh) [reply src/modules/10-common/components/ResourceCardList/ResourceCardList.tsx]: done

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good

**Review by litvinovstas — APPROVED** [review]

**Review by namanharness — APPROVED** [review]

**Pull request merged**